plugins/README

```diff
@@ -1,20 +1,14 @@
 README Concerning plugins for N+
 
 
-FOR EVERYBODY
-
-The plugin system is in its infant stage, a lot of things aren't implemented yet
-and a few things might be buggy. The provided plugins should be regarded as
-documentation and shouldn't be loaded unless you're willing to break things.
-
-
 FOR END USERS
 
-In order to get the plugins working you need to copy them to
-~/.nicotine/plugins, creating this directory if it doesn't exist yet. Plugins
-will be loaded upon startup, and can be reloaded with /reload. This reload
-command does not always completely reload the plugins due to the way Python
-works, in some cases a restart of N+ is needed to get new versions working.
+In order to load your own plugins you need to copy them to
```

Comment:
This actually should really be relevant more for developers than end users. End users should never have to do this. The plugins should ship within the deb and be correctly installed into the FHS.

Reply:
When I think of vanilla "end users" I don't normally think of people writing their own plugins.

Reply:
Good point.